src/vidpf.rs

```diff
@@ -337,6 +338,41 @@ impl<W: VidpfValue, const NONCE_SIZE: usize> Vidpf<W, NONCE_SIZE> {
 
         proof
     }
+
+    /// Evaluates multiple user reports.
+    pub fn multi_eval(
```

Comment:
I'm not sure of the use of this function, and whether it should be part of the public API or test-only. Generally, I don't think there's any computation that can be shared across multiple client reports, so it doesn't make much difference whether we loop over reports in the library or in the application. On the other hand, if this is anticipating batched verification, that would be useful to have in the library (though we wouldn't be able to make use of it within the VDAF abstraction).

Reply:
this is true, it's not actually used as is. 
I was trying to exemplify the use of the tree for caching. 
The tree will be used when mastic is in fact implemented.